examples/niswitch/controlling-an-individual-relay.py

```diff
@@ -1,16 +1,20 @@
-# Example communication with NI-SWITCH over gRPC
+# This example demonstrates how to control an individual relay on a module.
 #
 # The gRPC API is built from the C API.  NI-SWITCH documentation is installed with the driver at:
 # C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niSwitch\Documentation\English\SWITCH.chm
 #
+# A version of this .chm is available online at:
+# Link: https://zone.ni.com/reference/en-XX/help/375472H-01/
```

Comment:
This one looks a little bit older. I'm not saying I know if there is a new version or not, just wanted to make sure it is intentional.

Reply:
I think this is the latest.  These were the links sent to me by Curtis.  I'll follow up with him to ensure they're the latest